### PR TITLE
Refactor/orders

### PR DESCRIPTION
**Pull request by PhilippeGalvan**
Refactor order into dedicated objects.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,12 @@ If you want to create Binance account, here's [my referral link](https://www.bin
 ## Disclaimer
 Even if the scripts are easy to understand, errors exists that why the Issues section is important to report any bug that you may find. I recommend to use the [latest release](https://github.com/UPetit/python-binance-profit/releases/latest) because it has been tested multiple times compared to the other branches.
 ## Requirements
+- Python 3.7+
```
Comment (UPetit) [inline README.md]:
```suggestion
- Python 3.7+ (if you're new to Python, install it with [Anaconda](https://www.anaconda.com/)))
```

Comment (PhilippeGalvan) [reply README.md]:
```suggestion
- Python 3.7+, if you don't have it yet, you can install it:
  - Following [official instructions](https://docs.python.org/3/using/index.html)
  - With [Anaconda](https://www.anaconda.com/))

```

Comment (UPetit) [reply README.md]: Agreed!

**Comment by UPetit** [comment]
I've fixed some mistakes with my last 3 commits. I've tested the script and everything is working as expected 👌

**Comment by UPetit** [comment]
FYI: 
c2fddc2 has been tested and the OCOOrder cannot be executed because of the prices formats that are not valid.
Actually, `bought_price` is in the right format because it has been validated when the buy order has been executed earlier. However, the operations we're doing (`* (100 + profit)/100`) in order to create `price_profit` are probably adding too much zeros.

**Comment by UPetit** [comment]
I've tested the script as is: the OCO Order can't be validated. Still need to investigate the price formats

**Comment by UPetit** [comment]
Looks good to me. Thank you this great refactoring. Limit and Market orders have been tested successfully!

File: execute_orders.py [hunk 2]
```diff
@@ -25,49 +25,44 @@ def main(input_args: BaseModel) -> None:
...
     print(
-        f"=> Buy price: {get_formated_price(buy_price, symbol.price_decimal_precision)} "
+        f"=> Buy price: "
+        f"{get_formated_price(order_in_progress.info.price, symbol.price_decimal_precision)} "
         f"{symbol.quoteAsset}"
     )
     print(
         "=> Total price: "
-        f"{round(Decimal(buy_order['cummulativeQuoteQty']), symbol.price_decimal_precision)} "
+        f"{round(order_in_progress.info.cummulative_quote_quantity, symbol.price_decimal_precision)} "
```
Comment (UPetit) [inline execute_orders.py]:
```suggestion
        f"{get_formated_price(order_in_progress.info.cummulative_quote_quantity, symbol.price_decimal_precision)} "
```

**Comment by PhilippeGalvan** [comment]
@UPetit Thanks for the help during live XP sessions :) !

**Review by UPetit — APPROVED** [review]

**Pull request merged**